js/components/MilkdropWindow/index.js

```diff
@@ -103,6 +103,33 @@ class MilkdropWindow extends React.Component {
       }
     }
   }
+  _handleKeyboardInput(e) {
+    if (e.which === 32) {
```

Comment:
Could we add comments as to which keys these are? We do something similar here: https://github.com/captbaritone/webamp/blob/master/js/hotkeys.js#L64-L87